projects/plugins/jetpack/extensions/blocks/subscriptions/settings.scss

```diff
@@ -14,3 +14,21 @@
 	padding: 1em;
 	text-align: justify;
 }
+
+.edit-post-post-visibility {
```

Comment:
Do we need to modify core blocks, using core classes? These tend to be fragile long term.

If we must, I'd at least try adding custom classname prefixed with `jetpack-`.

Reply:
Done in d5ade0bd7fe4739f48d11cb685f6168f98dc5638

Noting here that it does not really matter as the CSS compiler will translate to 0 anyway.